cmd/create/cluster/cmd.go

```diff
@@ -1985,11 +1986,32 @@ func run(cmd *cobra.Command, _ []string) {
 	var availabilityZones []string
 	var subnets []*ec2.Subnet
 	if useExistingVPC || subnetsProvided {
-		initialSubnets, err := awsClient.GetSubnetIDs()
+		allSubnets, err := awsClient.GetSubnetIDs()
 		if err != nil {
 			r.Reporter.Errorf("Failed to get the list of subnets: %s", err)
 			os.Exit(1)
 		}
+		initialValidSubnets := []*ec2.Subnet{}
+		for _, subnet := range allSubnets {
+			hasRHManaged := false
+			for _, tag := range subnet.Tags {
+				switch *tag.Key {
+				case tags.RedHatManaged:
+					boolValue, err := strconv.ParseBool(*tag.Value)
+					if err != nil {
+						r.Reporter.Debugf("An error occurred whilst parsing bool value of '%s' tag, value '%s'", *tag.Key, *tag.Value)
+						boolValue = false
+					}
+					hasRHManaged = boolValue
```

Comment:
NP: Do we need `boolValue`? Couldn't we just assign `hasRHManaged` directly here?

Reply:
In the backend there is a similar check, I expect that if red-hat-managed is available it would be true. But who knows if some user decided to tag or change to false for some reason